lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -322,19 +322,14 @@ def _monitor(self, poll_interval):
                                   now.strftime("%H:%M:%S"))
                 self._poll_stats(datapath)
 
-    def _poll_stats(self, datapath):
+    def _poll_stats(self, datapath, cookie = 0, cookie_mask = 0):
```

Comment:
Here linter complains because we shouldn't have space between `=` so lets change this to
```
def _poll_stats(self, datapath, cookie=0, cookie_mask=0):
```